### PR TITLE
feat: Publish controller & telemetry-proxy to private marketplace ecr

### PR DESCRIPTION
**Pull request by nadav3396**

### CODE REVIEW

File: .github/workflows/build-test-release.yml [hunk 1]
```diff
@@ -557,3 +557,67 @@ jobs:
           artifacts: packages/*
           omitBody: true
           replacesArtifacts: true
+
+  publish-controller-image-marketplace-ownerd-ecr:
```
Comment (kenfinnigan) [inline .github/workflows/build-test-release.yml:561]: On both these jobs, is `ownerd` a typo? Should it be `owner` or `owned`?

Comment (nadav3396) [reply .github/workflows/build-test-release.yml:561]: yes, should be owned

Comment (nadav3396) [reply .github/workflows/build-test-release.yml]: fixed

File: .github/workflows/build-test-release.yml [hunk 2]
```diff
@@ -557,3 +557,67 @@ jobs:
...
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-east-1
+      - name: Login to Amazon ECR Private
+        id: login-ecr-private
+        uses: aws-actions/amazon-ecr-login@v1
+      - name: Build and push container image to Amazon ECR
+        env:
+          PROXY_IMG: 709825985650.dkr.ecr.us-east-1.amazonaws.com/lumigo/lumigo-kubernetes-telemetry-proxy:${{ needs.validate-release-increment.outputs.version }}
+        run: |
+          make docker-buildx-telemetry-proxy
```
Comment (kenfinnigan) [inline .github/workflows/build-test-release.yml:623]: Does the telemetry-proxy need a version for `make` like the previous job?

Comment (nadav3396) [reply .github/workflows/build-test-release.yml:623]: im not sure (honestly I copied it from the current publish jobs that publish to the public ecr we have and they works so I think it is OK like this)

File: .github/workflows/build-test-release.yml [hunk 3]
```diff
@@ -390,6 +390,14 @@ jobs:
       version: ${{ steps.export_version.outputs.version }}
 
   publish-controller-ecr-image:
+    strategy:
+      matrix:
+        ecr-registry: [ public.ecr.aws, 709825985650.dkr.ecr.us-east-1.amazonaws.com ]
```
Comment (mmanciop) [inline .github/workflows/build-test-release.yml:395]: I feel either this or `include` is redundant

**Review by moshe-shaham-lumigo — APPROVED** [review]

**Pull request merged**